restapi/api.go

```diff
@@ -37,6 +37,8 @@ func Router(db *gorm.DB, locationFinder location.LocationFinder) (http.Handler,
 		rest.GetHandler(r, "/story/{storyID}", api.ReturnStoryByID)
 		rest.GetHandler(r, "/authors/origin_countries", api.ReturnAllCountries)
 		rest.PostHandler(r, "/login", api.Login)
+
+		rest.GetHandler(r, "/client-tokens", api.ReturnClientTokens)
```

Comment:
We use `/authors/origin_countries` but here it is `/client-tokens`. We should standardize between `_` and `-`